### ISSUE TITLE
ConsentManagement requestBidsHook for video bug

### ISSUE THREAD

**Issue by ReinoutStevens** [prebid/Prebid.js#8300]
## Type of issue
This PR ( https://github.com/prebid/Prebid.js/issues/8185 ) breaks video's inside an iframe for us.

## Description
`getAdUnitSizes` does not return anything ( as it only works for banner units ), and then getting the width and height fails and throws an error.
Fixed this locally by changing the code to:

```
      if (Array.isArray(adUnits) && adUnits.length > 0) {
        let sizes = getAdUnitSizes(adUnits[0]);
        if (sizes.length > 0) {
          width = sizes[0][0];
          height = sizes[0][1];
        }
      }
```

## Steps to reproduce
Don't have an easily reproducible setup ( except our production and development domains where the issue was fixed )

Cleaned up stack trace of requesting 2 video ads inside an iframe:

```
Prebid INFO: Invoking pbjs.addAdUnits Arguments [Array(2), callee: (...), Symbol(Symbol.iterator): ƒ]0: Array(2)0: {code: 'midroll-skippable', mediaTypes: {…}, bids: Array(7), floors: {…}}1: {code: 'midroll-nonSkippable', mediaTypes: {…}, bids: Array(6), floors: {…}}length: 2[[Prototype]]: Array(0)callee: (...)length: 1Symbol(Symbol.iterator): ƒ values()get callee: ƒ ()set callee: ƒ ()[[Prototype]]: Object
Prebid MESSAGE: Emitting event for: addAdUnits
Prebid ERROR: Error processing command : Cannot read properties of undefined (reading '0') TypeError: Cannot read properties of undefined (reading '0')
    at prebid-v6.js:9:5010
    at Object.h (prebid-v6.js:9:5053)
    at f (prebid-v6.js:4:135217)
    at Object.f [as apply] (prebid-v6.js:4:135395)
```

Error points to `width = sizes[0][0];` in `requestBidsHook` in `consentManagement.js`

## Platform details
Prebid v6.19.0

## Other information

https://github.com/prebid/Prebid.js/issues/8185

**Comment by spormeon** [comment]
you getting some error like this?  I just tried 6.20.0 and i'm getting this:
![DevTools_-_www_cheftalk_com_ads_final_](https://user-images.githubusercontent.com/8503167/164222463-f0cbbe3c-e462-4037-9b93-2ec3b04e3c8b.png)

**Comment by dgirardi** [comment]
@EskelCz the fix for this was released in 6.20.1 - or are you saying this bug is still in master?

**Comment by EskelCz** [comment]
@dgirardi I was using 6.20.0, thanks for the info
